docs-client/src/lib/specification.tsx

```diff
@@ -162,6 +165,10 @@ export class Specification {
     return this.structsByName.get(name);
   }
 
+  public getVersion(): string | undefined {
```

Comment:
Is there any chance that the `version` is undefined?

Reply:
I checked it again, and it doesn't happen.
I'll change it.